[2023-07-30 22:05:10,582] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: dag_smtp_email_automatico.dag_envio scheduled__2023-03-09T00:00:00+00:00 [queued]>
[2023-07-30 22:05:10,644] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: dag_smtp_email_automatico.dag_envio scheduled__2023-03-09T00:00:00+00:00 [queued]>
[2023-07-30 22:05:10,645] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-07-30 22:05:10,645] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2023-07-30 22:05:10,645] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-07-30 22:05:10,845] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): dag_envio> on 2023-03-09 00:00:00+00:00
[2023-07-30 22:05:10,904] {standard_task_runner.py:52} INFO - Started process 1241 to run task
[2023-07-30 22:05:10,926] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'dag_smtp_email_automatico', 'dag_envio', 'scheduled__2023-03-09T00:00:00+00:00', '--job-id', '185', '--raw', '--subdir', 'DAGS_FOLDER/dag_smtp_email.py', '--cfg-path', '/tmp/tmpzsg0pu61', '--error-file', '/tmp/tmpejhduv14']
[2023-07-30 22:05:10,926] {standard_task_runner.py:80} INFO - Job 185: Subtask dag_envio
[2023-07-30 22:05:11,551] {task_command.py:371} INFO - Running <TaskInstance: dag_smtp_email_automatico.dag_envio scheduled__2023-03-09T00:00:00+00:00 [running]> on host b0e2d990d7ae
[2023-07-30 22:05:13,217] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Ariel Feldman
AIRFLOW_CTX_DAG_ID=dag_smtp_email_automatico
AIRFLOW_CTX_TASK_ID=dag_envio
AIRFLOW_CTX_EXECUTION_DATE=2023-03-09T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-03-09T00:00:00+00:00
[2023-07-30 22:05:14,737] {logging_mixin.py:115} INFO - (535, b'5.7.8 Username and Password not accepted. Learn more at\n5.7.8  https://support.google.com/mail/?p=BadCredentials 1-20020a4a0d01000000b0056082ad01desm3789779oob.14 - gsmtp')
[2023-07-30 22:05:14,738] {logging_mixin.py:115} INFO - Failure
[2023-07-30 22:05:14,738] {python.py:173} INFO - Done. Returned value was: None
[2023-07-30 22:05:14,796] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=dag_smtp_email_automatico, task_id=dag_envio, execution_date=20230309T000000, start_date=20230730T220510, end_date=20230730T220514
[2023-07-30 22:05:14,847] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-07-30 22:05:14,888] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
